### PR TITLE
Rc 5.2.0 Ihor

### PR DESCRIPTION
**Pull request by vik753**

### CODE REVIEW

**Review by AlekseyManetov — APPROVED** [review]

File: app/src/common/docs/BaseDocsBlock.module.scss [hunk 1]
```diff
@@ -22,6 +25,10 @@
 .width-wrapper {
     padding: 0 60px 60px;
     max-width: 960px;
+
+    h1, h2, h3, h4, h5, h6 {
+        color: var(--uui-text-dark);
+    }
```
Comment (siarhei-epam) [inline app/src/common/docs/BaseDocsBlock.module.scss:31]: There is no need to set the heading color here, it is already set in the typography

File: app/src/common/docs/DocExample.module.scss [hunk 2]
```diff
@@ -31,28 +31,30 @@
 }
 
 .preview-container {
-    background: $white;
-    border: 1px solid $gray40;
+    background: var(--uui-docs-background);
+    border: 1px solid var(--uui-secondary-20);
```
Comment (siarhei-epam) [inline app/src/common/docs/DocExample.module.scss:35]: looks like you need to use divider or control-border variable here

File: app/src/common/docs/DocExample.module.scss [hunk 3]
```diff
@@ -31,28 +31,30 @@
...
 }
 
 .code {
     font-size: 14px;
     line-height: 18px;
     padding: 24px;
     margin: 0;
-    border-top: 1px solid $gray40;
-    background-color: white;
+    border-top: 1px solid var(--uui-secondary-20);
```
Comment (siarhei-epam) [inline app/src/common/docs/DocExample.module.scss:46]: looks like you need to use divider or control-border variable here

File: app/src/common/docs/DocExample.module.scss [hunk 4]
```diff
@@ -31,28 +31,30 @@
...
+    background-color: var(--uui-surface);
+    color: var(--uui-text-primary);
```
Comment (siarhei-epam) [inline app/src/common/docs/DocExample.module.scss:48]: There is no need to set the heading color here, it is already set in the typography

File: app/src/common/docs/DocExample.module.scss [hunk 5]
```diff
@@ -31,28 +31,30 @@
...
     font-family: monospace;
     white-space: pre-wrap;
     word-wrap: normal;
     overflow: auto;
 
     /* Highly opinionated - make comments stand out and more readable // YakovZh */
     :global(.token):global(.comment) {
-        color: $gray40;
-        font-family: $font-sans;
+        color: var(--uui-text-primary);
```
Comment (siarhei-epam) [inline app/src/common/docs/DocExample.module.scss:56]: looks like you need to use text-disabled variable here

File: app/src/common/docs/DocExample.module.scss [hunk 6]
```diff
@@ -67,4 +69,9 @@
             opacity: 0.7;
         }
     }
+
+    :global(.token):global(.operator) {
+        background-color: var(--uui-surface);
+        color: var(--uui-warning-50);
```
Comment (siarhei-epam) [inline app/src/common/docs/DocExample.module.scss:75]: looks like we can create light and dark themes for Prism using our tokens

File: app/src/docs/_examples/dropdown/CloseFromBody.example.tsx [hunk 7]
```diff
@@ -1,17 +1,15 @@
...
+import { Button, Text, FlexRow, DropdownContainer, FlexCell } from '@epam/uui';
 import { IDropdownToggler } from '@epam/uui-core';
 
 export default function BasicDropdownExample() {
     const renderDropdownBody = (props: DropdownBodyProps) => {
         return (
             <DropdownContainer maxWidth={ 360 } vPadding="24" padding="18" { ...props }>
                 <FlexCell alignSelf="flex-start">
-                    <Text fontSize="18" lineHeight="24" color="gray90" font="museo-slab">
+                    <Text fontSize="18" lineHeight="24" color="secondary">
```
Comment (siarhei-epam) [inline app/src/docs/_examples/dropdown/CloseFromBody.example.tsx:12]: brand color is better for gray90

File: app/src/docs/_examples/dropdown/CloseOpenModifiers.example.tsx [hunk 8]
```diff
@@ -14,10 +12,10 @@ export default function CloseOpenModifiersExample() {
                     <Avatar size="48" alt="avatar" img="https://avatars.dicebear.com/api/human/avatar12.svg?background=%23EBEDF5&radius=50" />
 
                     <FlexCell width="100%">
-                        <Text cx={ css.text } lineHeight="24" fontSize="16" color="gray80" font="sans-semibold">
+                        <Text cx={ css.text } lineHeight="24" fontSize="16" color="secondary" font="semibold">
```
Comment (siarhei-epam) [inline app/src/docs/_examples/dropdown/CloseOpenModifiers.example.tsx:15]: primary color is better for gray80

File: app/src/docs/_examples/dropdown/CloseOpenModifiers.example.tsx [hunk 9]
```diff
@@ -14,10 +12,10 @@ export default function CloseOpenModifiersExample() {
...
                             John Doe
                         </Text>
-                        <Text cx={ css.text } lineHeight="18" fontSize="12" color="gray60">
+                        <Text cx={ css.text } lineHeight="18" fontSize="12" color="brand">
```
Comment (siarhei-epam) [inline app/src/docs/_examples/dropdown/CloseOpenModifiers.example.tsx:18]: tertiary color is better for gray60

File: app/src/docs/_examples/dropdown/DelayForOpenAndClose.example.tsx [hunk 10]
```diff
@@ -14,10 +12,10 @@ export default function DelayForOpenAndCloseExample() {
                     <Avatar size="48" alt="avatar" img={ `https://avatars.dicebear.com/api/human/${name}.svg?background=%23EBEDF5&radius=50` } />
 
                     <FlexCell width="100%">
-                        <Text cx={ css.text } lineHeight="24" fontSize="16" color="gray80" font="sans-semibold">
+                        <Text cx={ css.text } lineHeight="24" fontSize="16" color="secondary" font="semibold">
```
Comment (siarhei-epam) [inline app/src/docs/_examples/dropdown/DelayForOpenAndClose.example.tsx:15]: primary color is better for gray80

File: app/src/docs/_examples/dropdown/DelayForOpenAndClose.example.tsx [hunk 11]
```diff
@@ -14,10 +12,10 @@ export default function DelayForOpenAndCloseExample() {
...
                             {name}
                         </Text>
-                        <Text cx={ css.text } lineHeight="18" fontSize="12" color="gray60">
+                        <Text cx={ css.text } lineHeight="18" fontSize="12" color="brand">
```
Comment (siarhei-epam) [inline app/src/docs/_examples/dropdown/DelayForOpenAndClose.example.tsx:18]: tertiary color is better for gray60

File: app/src/docs/_examples/dropdown/HandleStateByYourself.example.tsx [hunk 12]
```diff
@@ -16,10 +14,10 @@ export default function BasicDropdownExample() {
                     <Avatar size="48" alt="avatar" img="https://avatars.dicebear.com/api/human/avatar12.svg?background=%23EBEDF5&radius=50" />
 
                     <FlexCell width="100%">
-                        <Text cx={ css.text } lineHeight="24" fontSize="16" color="gray80" font="sans-semibold">
+                        <Text cx={ css.text } lineHeight="24" fontSize="16" color="secondary" font="semibold">
                             John Doe
                         </Text>
-                        <Text cx={ css.text } lineHeight="18" fontSize="12" color="gray60">
+                        <Text cx={ css.text } lineHeight="18" fontSize="12" color="brand">
```
Comment (siarhei-epam) [inline app/src/docs/_examples/dropdown/HandleStateByYourself.example.tsx:20]: tertiary color is better for gray60

File: app/src/docs/_examples/dropdown/HandleStateByYourself.example.tsx [hunk 13]
```diff
@@ -16,10 +14,10 @@ export default function BasicDropdownExample() {
                     <Avatar size="48" alt="avatar" img="https://avatars.dicebear.com/api/human/avatar12.svg?background=%23EBEDF5&radius=50" />
 
                     <FlexCell width="100%">
-                        <Text cx={ css.text } lineHeight="24" fontSize="16" color="gray80" font="sans-semibold">
+                        <Text cx={ css.text } lineHeight="24" fontSize="16" color="secondary" font="semibold">
```
Comment (siarhei-epam) [inline app/src/docs/_examples/dropdown/HandleStateByYourself.example.tsx:17]: primary color is better for gray80

File: app/src/docs/_examples/tables/ArrayTable.example.tsx [hunk 14]
```diff
@@ -21,20 +21,20 @@ export default function ArrayDataTableExample() {
             {
                 key: 'id',
                 caption: 'Id',
-                render: (item) => <Text color="gray80">{item.id}</Text>,
+                render: (item) => <Text color="brand">{item.id}</Text>,
```
Comment (siarhei-epam) [inline app/src/docs/_examples/tables/ArrayTable.example.tsx:24]: Try to use this matching for text colors:
--uui-text-brand: var(--gray90);
    --uui-text-primary: var(--gray80);
    --uui-text-secondary: var(--gray70);
    --uui-text-tertiary: var(--gray60);
    --uui-text-disabled: var(--gray50);
    --uui-text-contrast: var(--gray5);
    
It cover all text colors for promo and can be used for other themes

**Review by siarhei-epam — APPROVED** [review]

**Pull request merged**